maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -161,16 +127,17 @@ public fun Marker(
     title: String? = null,
     visible: Boolean = true,
     zIndex: Float = 0.0f,
+    onDragEvent: ((DragState, LatLng) -> Unit)? = null,
```

Comment:
maybe just onDrag(...)?

Reply:
Not really opposed to it, except for English semantics, which are debatable. To me `onDrag` sounds like a singular complete drag action to a new position. Marker dragging is a sequence of events, at least 2, usually many more. `onDragEvent` carries a less succinct meaning to me, it could mean anything. `onDragSequenceEvent` would be better from that perspective, but is verbose.